### ISSUE TITLE
Quest issue around [26] Jin'Zil's Forest Magic

### ISSUE THREAD

**Issue by Gogo1951** [AeroScripts/QuestieDev#1978]
## Description

I got this LUA error when I accepted the quest on my 23 Undead Preist.

```
Quest accepted: Jin'Zil's Forest Magic

Date: 2020-05-25 12:50:58
ID: 2
Error occured in: Global
Count: 1
Message: ...terface\AddOns\Questie\Modules\QuestieReputation.lua line 26:
   attempt to perform arithmetic on a string value
Debug:
   (tail call): ?
   ...terface\AddOns\Questie\Modules\QuestieReputation.lua:26: Update()
   ...rface\AddOns\Questie\Modules\QuestieEventHandler.lua:310:
      ...rface\AddOns\Questie\Modules\QuestieEventHandler.lua:308
   [C]: ?
   ...ibDataBroker\CallbackHandler\CallbackHandler-1.0.lua:29:
      ...ibDataBroker\CallbackHandler\CallbackHandler-1.0.lua:25
   ...ibDataBroker\CallbackHandler\CallbackHandler-1.0.lua:64: Fire()
   ...rkInventory\Externals\Ace3\AceEvent\AceEvent-3.0.lua:120:
      ...rkInventory\Externals\Ace3\AceEvent\AceEvent-3.0.lua:119
Locals:
None
AddOns:
  Swatter, v8.2.6377 (SwimmingSeadragon)
  AdvancedInterfaceOptions, v1.3.4-3-g4eee160
  AlreadyKnown, v1.26
  ArkInventory, v30927
  ArkInventoryRules, v20000
  ArkInventorySearch, v20000
  AtlasLootClassic, vv1.5.1
  AtlasLootClassicData, vv1.5.1
  AtlasLootClassicDungeonsAndRaids, vv1.5.1
  AucAdvanced, v8.2.6430 (SwimmingSeadragon)
  AucFilterBasic, v8.2.6364 (SwimmingSeadragon)
  AucStatHistogram, v8.2.6366 (SwimmingSeadragon)
  AucStatiLevel, v8.2.6370 (SwimmingSeadragon)
  AucStatPurchased, v8.2.6367 (SwimmingSeadragon)
  AucStatSimple, v8.2.6399 (SwimmingSeadragon)
  AucStatStdDev, v8.2.6369 (SwimmingSeadragon)
  AucUtilFixAH, v8.2.6371 (SwimmingSeadragon)
  AutoDismount, v1.0.3
  BeanCounter, v8.2.6434 (SwimmingSeadragon)
  CEPGP, v
  CharacterStatsClassic, v3.5.3
  DBMCore, v1.13.45-50-g14169de
  DBMDefaultSkin, v
  DBMStatusBarTimers, v
  Details, v
  DetailsStreamer, v
  DetailsTinyThreat, v
  ElvUI, v1.24
  Enchantrix, v8.2.6428 (SwimmingSeadragon)
  EnchantrixBarker, v8.2.6469 (SwimmingSeadragon)
  FreeRefills, v116
  GatherMate2, v1.45.5-1-gab6216a
  GTFO, v4.55
  InFlight, v1.13.005
  InFlightLoad, v1.13.005
  Informant, v8.2.6374 (SwimmingSeadragon)
  InstanceResetAnnouncer, v1.0
  ItemTooltipProfessionIcons, v1.1.6
  LeatrixMaps, v1.13.64
  LeatrixPlus, v1.13.65.alpha.5
  MBB, v1.1.1
  Memoria, v8.2-2
  MonkeyBuddy, v
  MonkeyLibrary, v
  MonkeyQuest, v
  MonkeyQuestLog, v
  Outfitter, v1.2.0
  Pawn, v2.3.26
  QuestAnnounce, v8.0.2.1
  Questie, v5.10.3
  RecipeRadarClassic, v1.0.0.23
  RepSwitch, v8.3.0.21
  ReturnBuffTracker, v0.6.3
  SilverDragon, vv11302.0-3-g35ff885
  SilverDragonClassic, vv11302.0-3-g35ff885
  SimpleInterruptAnnounce, v1.0.0
  SimpleItemLevel, v1
  SimpleTauntAnnounce, v1.0.0
  SlideBar, v8.2.6375 (SwimmingSeadragon)
  SpeedyAutoLoot, v2.1.0-Alpha191024
  Spy, v1.0.28
  Stubby, v8.2.6376 (SwimmingSeadragon)
  Talented, vv191206
  TalentedData, v
  TankWarningsClassic, v1.1.9(BETA)
  TFTB, v1.10
  ThreatClassic2, v2.19
  TidyPlatesThreatPlates, v1.3.1
  TownsfolkTracker, v1.0.7
  TrackingEye, v2.0
  TradeSkillMaster, vv4.9.35
  VendorPrice, v1.3.5
  WeakAuras, v2.17.5-15-gbab5d5c
  WIM, v3.8.8
  BlizRuntimeLib_enUS v1.13.4.11304 <none>
  (ck=8a5)
```

## Wowhead link

* Jin'Zil's Forest Magic - Quest - World of Warcraft || https://classic.wowhead.com/quest=1058/jinzils-forest-magic

## Questie version

5.10.3

**Comment by drejjmit** [comment]
Fixed with b45c0236792c46d703e7eab9ca4f673312511333
It has nothing to do with the quest, you most likely gained a reputation level.

**Comment by Gogo1951** [comment]
Thanks!
